src/main/res/values/strings.xml

```diff
@@ -1162,7 +1162,7 @@
 	<string name="search_results_subreddit_only">Search on /r/%s</string>
 
 	<!-- 2020-02-17 -->
-	<string name="pref_network_tor_summary">When using an external browser or Custom Tabs, links will not be opened using Tor (unless your browser is configured to use Tor)</string>
+	<string name="pref_network_tor_summary">When using an external browser or custom tabs, links will not be opened using Tor (unless your web-browser is configured to use Tor)</string>
```

Comment:
Custom Tabs is deliberately capitalised to match Google's usage:

https://developers.google.com/web/android/custom-tabs

Also I've never seen "web browser" hyphenated :p

Reply:
From the brilliant minds that wrote "If you are getting started with Custom Tabs, _checkout_ the Implementation Guide".
Should it be "external file-browser" or "external web-browser" btw?

Reply:
Haha :smile: Even so, let's keep it capitalised as it makes it clearer that it's referring to the Android feature. In fact I had someone message me a few weeks ago asking what it meant, so maybe best as "Android Custom Tabs" for maximum clarity.

It should be "web browser" (no hyphen please)